test/src/test/java/hudson/ClassicPluginStrategyTest.java

```diff
@@ -70,7 +70,9 @@ protected Hudson newHudson() throws Exception {
             }
         };
         setPluginManager(pluginManager);
-        return new Hudson(home, createWebServer(), pluginManager);
+        // TODO needs triage
+        //return new Hudson(home, createWebServer(), pluginManager);
+        return new Hudson(home, null, pluginManager);
```

Comment:
I haven't even begun to try and get the test harness up and running, but needless to say there will be a lot of changes required there as well.